CONTRIBUTING.md

```diff
@@ -0,0 +1,177 @@
+# CONTRIBUTING
+
+## Introduction
+
+This document explains how to contribute to Invidious.
+
+Each section below describes a different way to contribute, based on your skills and experience
+with Invidious. Here is a summary:
+
+* [Issues](#issues): You are a regular user, and want to report a bug, ask for a new feature or for
+  an existing feature to be improved.
+
+* [Translations](#translations): You speak English and you're fluent in another language, and want
+  to help us reach more users around the globe.
+
+* [Hosting an instance](#hosting-an-instance): You're a sysadmin and have some time and server
+  ressources to spare, with some prior experience with invidious.
+
+* [Testing](#testing): You're an advanced user, who is already running their own instance, is not
+  worried of compiling Invidious from source, and wants to help us test bug fixes or new features.
+
+* [Development](#development): You're a developper and want to contribute code, or help with
+  reviewing the code written by others.
+
+
+
+## Issues
+
+We use the [Github issue tracker](https://github.com/iv-org/invidious/issues) to track and manage
+bug reports, feature requests and improvements.
+
+**Note: Before opening any kind of issue, make sure to search on the tracker
```

Comment:
I would even add something along the lines of "If you spot a duplicate issue, feel free to point it out"
This can help with reducing duplicates, where users who create duplicates can close them when they are made aware of the original issue